src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,33 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet("/cars/addDrivers")
```

Comment:
The previous url was good (@WebServlet("/cars/drivers/add")), suggestion was about jsp naming)